lib/public/utilities/fetch/RemoteDataSource.js

```diff
@@ -10,35 +10,39 @@
  *  granted to it by virtue of its status as an Intergovernmental Organization
  *  or submit itself to any jurisdiction.
  */
-import { Observable, RemoteData } from '/js/src/index.js';
+import { RemoteData } from '/js/src/index.js';
 import { getRemoteData } from './getRemoteData.js';
 import { ObservableData } from '../ObservableData.js';
+import { DataSource } from '../DataSource.js';
 
 /**
- * Service class providing observable data fetching process
+ * Class to abstract notifying from data changes
+ * @template T - type of response sent by server
```

Comment:
Please stop mentioning server from the fronted. TBH the name I don't like the name `RemoteData` because it presumes too much about the data, for example it could simply be data loaded from disk or parsing of input's file. Anyway, sticking to conventions here, T is simply `type of the remote data`

Reply:
applied